cerner/terra-framework
# Fix escape key disabling focus trap in notification dialog

### Summary
Fixes an issue where pressing the escape key while the notification dialog was open would disable the focus trap allowing users to shift focus outside of the notification dialog.

### Steps to test
* Go to [Notification Dialogue](https://terra-framework-deploye-pr-683.herokuapp.com/#/components/terra-notification-dialog/notification-dialog/notification-dialog). 
* In the ' Notification Dialog on Modal Manager' example, click Disclose> Trigger NotificationDialog.
* Press Esc once.
* Press tab or shift+tab repeatedly

Focus should stay within notification dialog.
